packages/wifi-signal-strength/default.nix

```diff
@@ -13,8 +13,12 @@
     pkgs.writeShellScript
     "wifi-signal-strength"
     ''
+      trap "${pkgs.openssh}/bin/ssh -q -S /tmp/nmcli_socket -O exit ghaf@${netvm_address}" EXIT
```

Comment:
https://github.com/tiiuae/ghaf/pull/549/files#diff-0dd9147b12941844c66d8519a708fb64e934748abb257ff28c09de8bec9dc667R47

Do you not already do that here? 

If I am not mistaken EXIT only triggers on normal exit (0), which only occurs after the line listed above. So could we replace line 47 with an exit 0 to trigger the trap?

Reply:
Yes, you are right. The script will be clearer with it, thanks!